index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">    
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <div class="logo">
+            <a href="">Alexander</a>
+        </div>
+        <ul class="top-navigation">
+            <li class="navigation-item">
+                <a href="">Portfolio</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">About</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">Contact</a>
+            </li>
+        </ul>
+        <div class="menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+        </div>
+    </nav>
+
+    <main class="main" >
+        <div class="headline">
+            <div class="social-media">
+                <a href=""><img src="./ressources/icons/twitter.svg" alt="logo_twitter" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/github.svg" alt="logo_github" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/Linkedin.svg" alt="logo_Linkedin" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/angel.svg" alt="logo_angel" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/medium.svg" alt="logo_medium" 
+                    class="logo-social">
+                </a>
+            </div>
+            <div class="text-headline">
+                <p class="primery-text">
+                    Hey There. <br>I'm Alex
+                </p>
+                <p class="supporting-text">
+                    I’am a Software Developer
+                </p>
+                <p class="description">
+                    I can help you build a product , 
+                    feature or website Look through some of my work and experience! I
+                    f you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            
```

Comment:
- [ ] Kindly make sure that your design (style, margins, fonts family, background images) matches that of the figma template as much as possible.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/164451172-fb083169-b404-4856-a66a-083387832cbe.PNG)

Figma template
![figma](https://user-images.githubusercontent.com/85095702/164451271-ba1c6353-546f-4a05-a425-ac26bb9edb64.PNG)

Reply:
done 👍 i have just forgeted this